core/src/Streamly/Internal/Data/Stream/Serial.hs

```diff
@@ -126,5 +126,5 @@ map f = Stream.mapM (return . f)
 -- /Pre-release/
 --
 {-# INLINE unfoldrM #-}
-unfoldrM :: Monad m => (b -> m (Maybe (a, b))) -> b -> Stream m a
+unfoldrM :: Monad m => (b -> m (Maybe (a, b))) -> b -> SerialT m a
```

Comment:
Now that we have `unfoldrM` in the Data.Stream module we can just directly export that from this module instead of redefining it here.